cli/js/ops/fs/chmod.ts

```diff
@@ -1,10 +1,17 @@
 // Copyright 2018-2020 the Deno authors. All rights reserved. MIT license.
 import { sendSync, sendAsync } from "../dispatch_json.ts";
+import { pathFromURL } from "../../util.ts";
 
-export function chmodSync(path: string, mode: number): void {
+export function chmodSync(path: string | URL, mode: number): void {
+  if (path instanceof URL) {
+    path = pathFromURL(path);
+  }
```

Comment:
You could move the condition inside the function to have
```suggestion
  path = pathFromMaybeURL(path);
```
(no actual need to change the function name, though).